### PR TITLE
Bump versions to 5.4.1

### PR DESCRIPTION
**Pull request by jeqo**
Aligning with latest cp version.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -61,13 +66,13 @@ jar {
...
 jib {
+  from.image = 'confluentinc/cp-kafka:${cpVersion}'
   to.image = 'cnfldemos/cp-demo-kstreams:0.0.5'
-  from.image = 'confluentinc/cp-kafka:5.4.0'
   container {
     entrypoint = '/app/start.sh'
     useCurrentTimestamp=true
   }
-  extraDirectory { 
+  extraDirectories {
```
Comment (jeqo) [inline build.gradle:75]: piggybacking update of this deprecated config

**Comment by rspurgeon** [comment]
Thanks a bunch for the PR @jeqo !  Could you bump `version` to `0.0.6` please?

File: build.gradle [hunk 2]
```diff
@@ -15,10 +14,15 @@ plugins {
 }
 
 group = 'io.confluent'
-version = '0.0.4'
```
Comment (jeqo) [inline build.gradle:18]: seems that versions drifted between project and docker image

**Comment by jeqo** [comment]
bumped :)

File: build.gradle (hunk 2 above)
Comment (rspurgeon) [reply build.gradle:18]: I agree, we have a task to transition this to maven so we can leverage some automation across the rest of our projects.  I still very much appreciate your contribution!

**Comment by rspurgeon** [comment]
Published: https://hub.docker.com/layers/cnfldemos/cp-demo-kstreams/0.0.6/images/sha256-1b3a6f98f92c70fc974651c4a2e136641e13d8686de5947638497b3b190d5310?context=explore

Thanks again @jeqo

**Pull request merged**